### ISSUE TITLE
Sort elements by name too?

### ISSUE THREAD

**Issue by er1c** [therealmarv/logic-xml-diff#2]
Curious if you can sort the elements by name too and not just the attributes.

**Comment by therealmarv** [comment]
yes this is also possible. I will think of how to implement that. But for many use cases sorting elements by name is also changing the structure/tree of a XML.

**Comment by er1c** [comment]
The usage scenario I was looking into was "sorting" two  XSD files  - effectively need to sort the `xs:element` by name:

    <xs:element name="getVendorIds2" nillable="true" type="tns:vendorIds2Request"/>
    <xs:element name="getVendorIds2Response" nillable="true" type="tns:vendorIds2Response"/>

And then by `xs:complexType` by name/etc

  <xs:complexType name="amBrandAddressRecord" final="extension restriction">
